documentation/book/assembly-metrics-kafka.adoc

```diff
@@ -3,21 +3,14 @@
 // assembly-metrics-setup.adoc
 [id='assembly-metrics-kafka-{context}']
 
-= Kafka metrics configuration
+= JMX metrics
```

Comment:
I would not call it JMX metrics. JMX is a protocol which we don't support here. I think we should call it either Kafka metrics as before (which will be understandable to anyone without any experience with metrics on Kubernetes). Or _Prometheus metrics_ which is what the users will actually use.

Reply:
Okay. Back to Kafka Metrics.